### ISSUE TITLE
Minimal header web component - Accessibility review

### ISSUE THREAD

**Issue by caw310** [department-of-veterans-affairs/vets-design-system-documentation#2100]
## Issue Description
We need to conduct a full accessibility review of the Minimal header web component. The component can be viewed in Storybook here: [insert Chromatic link here]
See development ticket: https://github.com/department-of-veterans-affairs/vets-design-system-documentation/issues/2030

## Tasks
- [ ] Full accessibility review of the component is completed
- [ ] Add descriptions of any issues found to this ticket
    - [ ] If any problems are found, add screen recordings and/or screenshots to comments on this ticket that show the problem.
- [ ] If necessary, discuss any accessibility issues found with the engineer who developed the component
- [ ] Re-review any fixes the developer makes to the component

## Acceptance Criteria
- [ ] Component has been reviewed for accessibility issues
- [ ] Any accessibility issues have been addressed

**Comment by caw310** [comment]
Moving this to the next sprint as development is not yet done.

**Comment by caw310** [comment]
Could not be worked on in sprint 37 as planned as development was ongoing and is still not complete.

**Comment by rsmithadhoc** [comment]
- **Must:** Missing yellow focus outline on the VA logo
- **Must:** On the VA Logo link, there should be some text for screen readers. Adding an `aria-label` to the link should be sufficient.
- **Must**: There were some issues with the crisis line modal, I don't see these issues on the main header in production.
    - You can tab outside of the modal to the VA logo link. This also means that focus is not trapped inside the modal, as it should be (see production's crisis modal).
    - The `aria-label` for the close button is "Close We’re here anytime, day or night – 24/7". I think "Close Modal" or "Close Crisis Line Modal" would be better.
- Not accessibility, but a few design things I noticed (on Mac + Firefox)
    - Crisis line icon is 1px off.
    - VA icon is not centered on the blue rectangle.

**Comment by caw310** [comment]
@rsmithadhoc would you write up a ticket with your findings and the Musts?

**Comment by rsmithadhoc** [comment]
> @rsmithadhoc would you write up a ticket with your findings and the Musts?
Yes, I will.

**Comment by rsmithadhoc** [comment]
@caw310 Issue created here: https://app.zenhub.com/workspaces/platform-design-system-5f8de67192551b0012ebb802/issues/gh/department-of-veterans-affairs/vets-design-system-documentation/2220
